src/main/java/com/kruize/query/PrometheusQuery.java

```diff
@@ -35,42 +37,72 @@ public static PrometheusQuery getInstance()
     }
 
     @Override
-    public String getCpuQuery(String podName)
+    public String getCpuQuery(String instanceName)
     {
-        return "rate(container_cpu_usage_seconds_total{" +
-                "pod_name=~\"" + podName + "\",container_name!=\"POD\"}[1m])";
+        if(DeploymentInfo.getClusterType().toUpperCase().equals("DOCKER")) {
```

Comment:
How can we avoid checking for "Docker" every time ?
Please review all `if` statements for coding standard.